### PR TITLE
Implement Bithumb

### PR DESCRIPTION
**Pull request by Swingcloud**

### CODE REVIEW

File: lib/cryptoexchange/exchanges/bithumb/services/market.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        def ticker_url(market_pair)
+          "#{Cryptoexchange::Exchanges::Bithumb::Market::API_URL}/public/ticker/#{market_pair.base}"
+        end
+
+        def adapt(output, market_pair)
+          ticker = Bithumb::Models::Ticker.new
+          ticker.base = market_pair.base
+          ticker.target = market_pair.target
+          ticker.market = Bithumb::Market::NAME
+          ticker.ask = output['data']['sell_price'] ? BigDecimal.new(output['data']['sell_price'].to_s) : nil
```
Comment (dominicwong617) [inline lib/cryptoexchange/exchanges/bithumb/services/market.rb]: @Swingcloud  can we use `NumericHelper.to_d` from `master`?

**Comment by dominicwong617** [comment]
@Swingcloud the test is failing because of `available_exchanges`

**Comment by Swingcloud** [comment]
yeah I fix this fail at  #35

**Comment by dominicwong617** [comment]
@Swingcloud I merged #35 , can you rebase it to this branch and get the build pass? Thanks

**Comment by Swingcloud** [comment]
sure thing

**Pull request merged**